scripts/us_census/pep/population_estimate_by_race/preprocess.py

```diff
@@ -760,103 +891,149 @@ def _transform_data(self, df: pd.DataFrame) -> None:
+                                        index=False)
+            #Added by Shamim to resolve 2020 inconsistent data remove
+            df_national_state_2000 = df_national_state_2000.drop_duplicates(
+                subset=['geo_ID', 'Year'], keep='last')
+            df_national_state_2000.to_csv(os.path.join(
+                self.cleaned_csv_file_path,
+                "USA_Population_Count_by_Race_National_state_2000.csv"),
+                                          index=False)
+        except Exception as e:
+            logging.error("error processing file")
+            return False
+        return True
 
     def process(self):
         """
         This is main method to iterate on each file,
         calls defined methods to clean, generate final
         cleaned CSV file, MCF file and TMCF file.
         """
+        #input_path = _FLAGS.input_path
+        ip_files = os.listdir(self.input_path)
+        self.input_files = [
+            self.input_path + os.sep + file for file in ip_files
+        ]
+        processed_count = 0
+        total_files_to_process = len(self.input_files)
+        logging.info(f"No of files to be processed {len(self.input_files)}")
         for file in self.input_files:
+            print(file)
+            if 'USCountywv90.txt' in file:
+                pass
             df = self._load_data(file)
-            self._transform_data(df)
-        name = "USA_Population_Count_by_Race_before_2000"
-        generator_df=pd.read_csv\
-            (os.path.join(self.cleaned_csv_file_path,
-                "USA_Population_Count_by_Race_before_2000.csv"))
-        self._generate_mcf(generator_df.columns, name)
-        self._generate_tmcf(generator_df.columns, name)
-
-        name = "USA_Population_Count_by_Race_county_after_2000"
-        generator_df=pd.read_csv\
-            (os.path.join(self.cleaned_csv_file_path,
-            "USA_Population_Count_by_Race_county_after_2000.csv"))
-        self._generate_mcf(generator_df.columns, name)
-        self._generate_tmcf(generator_df.columns, name)
-
-        name = "USA_Population_Count_by_Race_National_state_2000"
-        generator_df=pd.read_csv\
-            (os.path.join(self.cleaned_csv_file_path,
-            "USA_Population_Count_by_Race_National_state_2000.csv"))
-        self._generate_mcf(generator_df.columns, name)
-        self._generate_tmcf(generator_df.columns, name)
+            result = self._transform_data(df)
+            if result:
+                processed_count += 1
+        logging.info(f"No of files processed {processed_count}")
+        if processed_count == total_files_to_process & total_files_to_process > 0:
+
+            name = "USA_Population_Count_by_Race_before_2000"
+            generator_df=pd.read_csv\
+                (os.path.join(self.cleaned_csv_file_path,
+                    "USA_Population_Count_by_Race_before_2000.csv"))
+            #print("Before delete ", generator_df.shape)
```

Comment:
Can use logging.debug("...") for debug statements.

They will be shown by dong the following in main()
  logging.verbosity(1)